mne/forward/_lead_dots.py

```diff
@@ -139,7 +145,26 @@ def _comp_sum_eeg(beta, ctheta, lut_fun, n_fact):
 
 
 def _comp_sums_meg(beta, ctheta, lut_fun, n_fact, volume_integral):
-    """Lead field dot products using Legendre polynomial (P_n) series"""
+    """Lead field dot products using Legendre polynomial (P_n) series.
+
+    Parameters
+    ----------
+    beta : array, shape (n_points * n_points, 1)
+        Coefficients of the integration.
+    ctheta : array, shape (n_points * n_points, 1)
+        Cosine of the angle between the sensor integration points.
+    lut_fun : object
```

Comment:
that's the best I could figure out :) maybe @Eric89GXL can give a hand with the docs here.